docs/decex/xdm.md

```diff
@@ -286,18 +287,18 @@ Before sending any messages, domain needs to have an channel open with the `dst_
 
 ### Close Channel
 
-Any domain of either end of the open channel can close the channel:
+Any domain of either end of an `Open` or `Initiated` channel can close the channel:
```

Comment:
This is not done yet but will be handled in next PR. Currently just Open channels can be closed. FYI

Reply:
Understood. The spec is prescriptive here of what should be happenning.